### ISSUE TITLE
Feedback within the survey at the end of a test...

### ISSUE THREAD

**Issue by HW71** [rainforestapp/tester-chrome-extension#428]
Hi all,

in the recent news Kristen reminds us to report "No" in case of poorly written instructions + additionally to provide feedback using the survey at the end of the test in order to give details.

The problem I see with this "survey": the input field used in this form offers just one line to enter feedback, right? At least this was the case when I used that survey the last time.

=> Would it be possible to offer an ordinary TestArea element instead so we see the whole text making it finally easier for us to edit the feedback? 

TIA & Cheers, Erik

**Comment by nerdatwork** [comment]
Even though you see 1 line you can actually enter a whole paragraph of text in it. Try it and you will be pleasantly surprised :)

**Comment by marius11** [comment]
> you will be pleasantly surprised :)

It's not pleasant reading a paragraph in a text box that has a single line and a small length.

**Comment by JPierreS** [comment]
Indeed I agree with @marius11 - Whilst you can write paragraphs into 1 line, it would be much cleaner to provide a text box.  However I think the intentions for this survey is not to write paragraphs - The main part is the survey, you are rating the virtual VM, the test instructions and the tester interface which is self explanatory.  You can send the right message with a small note.

For example some tests I rate a 1 and in notes I write could write 1 line to describe my vote, here are some examples:

"Too many steps into 1 it is hard to follow"
"Instructions are confusing"
"Instructions do not follow our tester rules for matching quotes"

In this case long paragraphs are not needed for most cases.

**Comment by HW71** [comment]
Hi all,

of course one can write "Steps are confusing". For that a single line field is absolute sufficient. But at least when I write test cases @ work and send them to review I expect a little bit more than that.

On the other hand the survey and the related feedback is most probably intended for RF (while the comment we leave after clicking "No" gets read by the client). So I guess both feedbacks should differ from content perspective and detailedness.

Well, by the end of the day this all is digging in the dark. If RF is happy with "one-liner" like this then I'll by happy, too. :-) Maybe a RF official could comment / clarify => right now it looks like I could close this one because of "works as designed"... :-)

**Comment by JPierreS** [comment]
These are not read by clients directly, but rather by one of the CSMs who work with the clients  :)
I guess having a bigger text box would be good for some situations that warrant more explanations.
I guess this would not hurt.  :)

**Comment by HW71** [comment]
Well, since this enhancement request is not related to the extension and no additional comments have been posted for several weeks I'll close this one.
